dist/images/Dockerfile

```diff
@@ -9,6 +9,9 @@ COPY grace_stop_ovn_controller /usr/share/ovn/scripts/grace_stop_ovn_controller
 
 WORKDIR /kube-ovn
 
+RUN rm -f /usr/bin/nc &&\
```

Comment:
we should remove ncat from base

Reply:
Yes. The nc cmd is installed in nmap packet, so I delete the nmap packet in the last PR https://github.com/kubeovn/kube-ovn/pull/2141
But the nc cmd still exists because the ncat is installed in dockerfile.base.

The jira about security only reports cmd about nc and netcat, both are soft link from ncat.
<img width="534" alt="image" src="https://user-images.githubusercontent.com/67130442/206945444-abede5dc-78dd-472f-8ce0-d1f26dbf688d.png">

So if we can keep ncat and delete the soft link cmd? It will be convennient if we want to use nc in the pod.
Or It's better to delete ncat totally?

Reply:
ok